src/main/webapp/resources/js/apis/users/groups.js

```diff
@@ -12,3 +12,59 @@ const BASE_URL = setBaseUrl("/ajax/user-groups");
 export function deleteUserGroup(id) {
   return axios.delete(`${BASE_URL}?id=${id}`).then(({ data }) => data);
 }
+
+/**
+ * Get details about a specific user group
+ *
+ * @param {number} id for a user group
+ * @returns {Promise<AxiosResponse<any>>}
+ */
+export function getUserGroupDetails(id) {
+  return axios.get(`${BASE_URL}/${id}`).then(({ data }) => data);
+}
+
+/**
+ * Update the User Group fields
+ * @param {number} id for the user groupd
+ * @param {string} field to update
+ * @param {string} value to set
+ * @returns {Promise<AxiosResponse<any>>}
+ */
+export function updateUserGroupDetails({ id, field, value }) {
+  return axios.put(`${BASE_URL}/${id}/update `, {
+    field,
+    value,
+  });
+}
+
+export function getUserGroupRoles() {
```

Comment:
Add comments to this function and the ones below

Reply:
Updated in ac0e9bf28b8bcf0ed4d214cb35ff57d88fc2dbe3